.github/workflows/build.yml

```diff
@@ -12,18 +12,18 @@ on:
 jobs:
```

Comment:
Validating the [Action artifacts](https://github.com/ni/VireoSDK/actions/runs/8854607584) by [running that tgz on the ASW CI](https://github.com/ni/VireoSDK/blob/673eb0dea01418f6ca20e6a2a53528a941eef7ca/docs/Workflow.md#testing-vireojs-on-a-different-machine--ci) with a [draft PR](https://dev.azure.com/ni/DevCentral/_git/ASW/pullrequest/702174). Also triggered the full [VireoTest squad tests](https://ni.visualstudio.com/DevCentral/_wiki/wikis/AppCentral.wiki/2445/How-to-update-Vireo-in-NXG?anchor=trigger-the-web-module-squad-tests). Waiting for the build to complete.

Only tested the npm package but as a follow-up will create a [hotfix tag](https://github.com/ni/VireoSDK/blob/673eb0dea01418f6ca20e6a2a53528a941eef7ca/docs/Workflow.md#creating-hotfix-releases) to test the full release process.

Reply:
Builds passed! Let's go!